.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 /dist
 /build
 /lib
+.vscode
```

Comment:
This file is attached on part of the project and is specific to your setup. You should add this in to your global git ignore.